src/network/networkd-network.h

```diff
@@ -153,6 +153,7 @@ struct Network {
         bool dhcp_use_timezone;
         bool dhcp_use_hostname;
         bool dhcp_use_6rd;
+        bool dhcp_allow_ipv6_only_preferred;
```

Comment:
Please make the variable name consistent with the setting name: `_preferred` -> `_mode`.